apps/chat/src/components/Promptbar/components/PromptFolders.tsx

```diff
@@ -120,7 +122,12 @@ const PromptFolderTemplate = ({
 
   const handleFolderClick = useCallback(
     (folderId: string) => {
-      dispatch(UIActions.toggleFolder({ id: folderId }));
+      dispatch(
+        UIActions.toggleFolder({
```

Comment:
@mikitabut , because it conversations I implemented lazy-loading(i.e. we load data for folder if need) but in prompts we upload all folder structure initially